src/platform/macos.rs

```diff
@@ -131,6 +131,7 @@ pub trait WindowBuilderExtMacOS {
     /// Build window with `resizeIncrements` property. Values must not be 0.
     fn with_resize_increments(self, increments: LogicalSize<f64>) -> WindowBuilder;
     fn with_disallow_hidpi(self, disallow_hidpi: bool) -> WindowBuilder;
+    fn with_ignore_alt_modifier(self, ignore_alt_modifier: bool) -> WindowBuilder;
```

Comment:
I feel like it should be `with_option_as_alt`, since it's more clear to the users. Getting inspiration from other apps could be a good idea.

Reply:
Yes, we should make this flag as descriptive as possible.

Reply:
This should still be changed.

Reply:
Okay, I'll make the change to `with_option_as_alt`